src/Build/Evaluation/Evaluator.cs

```diff
@@ -200,6 +200,16 @@ internal class Evaluator<P, I, M, D>
 
         private readonly EvaluationProfiler _evaluationProfiler;
 
+        /// <summary>
+        /// Keeps track of all imported projects with the newest first.
```

Comment:
nit: newest as in latest found by the evaluator, or newest as in most recently written to?

Reply:
Though I guess the DateTime in the type says it all :)

---
In reply to: [174532847](https://github.com/Microsoft/msbuild/pull/3038#discussion_r174532847) [](ancestors = 174532847)